src/content/fr/2019/javascript.md

```diff
@@ -0,0 +1,433 @@
+
+<figure>
+   <a href="/static/images/2019/javascript/fig3.png">
+      <img src="/static/images/2019/javascript/fig3.png" alt="Figure 3. Temps de traitement du fil d’exécution principal V8 par appareil." aria-labelledby="fig3-caption" aria-describedby="fig3-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=924000517&format=interactive">
+   </a>
+   <div id="fig3-description" class="visually-hidden">Un diagramme à barres indiquant que 141&nbsp;ms&#8239;/&#8239;377&nbsp;ms de temps de traitement sont utilisés dans le percentile p10 sur le bureau et le mobile respectivement, 352&#8239;/&#8239;988&nbsp;ms pour p25, 849&#8239;/&#8239;2&#8239;437&nbsp;ms pour p50, 1&#8239;850&#8239;/&#8239;5&#8239;518&nbsp;ms pour p75, et 3&#8239;543&#8239;/&#8239;10&#8239;735&nbsp;ms pour p90.</div>
+   <figcaption id="fig3-caption">Figure 3. Temps de traitement du fil d’exécution principal V8 par appareil.</figcaption>
+</figure>
+
+À chaque centile, les délais de traitement sont plus longs pour les pages web testées sur mobiles que pour les pages testées sur ordinateurs de bureau. Le temps total médian du fil d’exécution principal sur ordinateurs de bureau est de 849&nbsp;ms, tandis que sur mobiles, il est plus important&nbsp;: 2&#8239;437&nbsp;ms.
+
+Bien que ces données montrent combien de temps il peut falloir à un appareil mobile pour traiter JavaScript par rapport à un ordinateur de bureau plus puissant, les appareils mobiles varient également en termes de puissance de calcul. Le tableau suivant montre comment les temps de traitement d’une seule page web peuvent varier considérablement selon la catégorie d’appareil mobile.
+
+<figure>
+   <a href="/static/images/2019/javascript/js-processing-reddit.png">
+      <img src="/static/images/2019/javascript/js-processing-reddit.png" alt="Délais de traitement de JavaScript pour Reddit.com." aria-labelledby="fig4-caption" aria-describedby="fig4-description" width="600" height="363">
+   </a>
+   <div id="fig4-description" class="visually-hidden">Diagramme à barres montrant 3 appareils différents&nbsp;: en haut, un Pixel 3 présentant une petite portion à la fois sur le fil principal et le fil secondaire de moins de 400&nbsp;ms. Dans le cas du Moto G4, cela représente environ 900&nbsp;ms sur le fil principal et 300&nbsp;ms sur le fil secondaire. La dernière barre est celle d’un Alcatel 1X 5059D avec plus de 2&#8239;000&nbsp;ms sur le fil principal et plus de 500&nbsp;ms sur le fil secondaire.</div>
+   <figcaption id="fig4-caption">Figure 4. Délais de traitement de JavaScript pour Reddit.com, issus de <a href="https://v8.dev/blog/cost-of-javascript-2019">The cost of JavaScript in 2019 (en anglais)</a>.</figcaption>
```

Comment:
Add a `lang="en"`?:

```suggestion
   <figcaption id="fig4-caption">Figure 4. Délais de traitement de JavaScript pour Reddit.com, issus de <a href="https://v8.dev/blog/cost-of-javascript-2019"><i lang="en">The cost of JavaScript in 2019</i> (en anglais)</a>.</figcaption>
```

Note can't put it on the `<a href...>` element as not all of it is in English.